build.py

```diff
@@ -1108,14 +1094,7 @@ def create_dockerfile_linux(
 RUN python3 -m pip install --upgrade pip \\
       && pip3 install transformers
 
-# Drop the static libs
-RUN ARCH="$(uname -i)" \\
-      && rm -f ${TRT_ROOT}/targets/${ARCH}-linux-gnu/lib/libnvinfer*.a \\
-          ${TRT_ROOT}/targets/${ARCH}-linux-gnu/lib/libnvonnxparser_*.a
-
```

Comment:
Why removing these lines? Is dropping libs causing some issues?

Reply:
Removing of those libs doesn't make our container image smaller if that's what is it designed for.
Is the removing of those libs has any other logical purpose?

Reply:
Thought it would reduce the container size.. Thanks for the explanation.